### PR TITLE
Ux changes

### PR DESCRIPTION
**Pull request by pranshudobhal**

### CODE REVIEW

File: components/Footer/Footer.js [hunk 1]
```diff
@@ -1,26 +1,114 @@
...
+          <div className="footer-icons-container">
+            <FooterIconLink href="https://www.instagram.com/neogcamp" imgSrc="/images/instagram.svg" />
+            <FooterIconLink href="https://twitter.com/neogcamp" imgSrc="/images/twitter.svg" />
+            <FooterIconLink href="https://www.youtube.com/watch?v=Ezk2AwqgS9Q&list=PLzvhQUIpvvuj5KPnyPyWsvgyzNkX_ACPA" imgSrc="/images/youtube.svg" />
+            <FooterIconLink href="https://www.linkedin.com/company/neog-camp/" imgSrc="/images/linkedin.svg" />
+            <FooterIconLink href="https://letters.neog.camp/" imgSrc="/images/substack.svg" />
+            <FooterIconLink href="https://www.facebook.com/neogcamp" imgSrc="/images/facebook.svg" />
+            <FooterIconLink href="https://t.me/teamtanay" imgSrc="/images/telegram.svg" />
+          </div>
+          {/* <div className="footer links-container">
```
Comment (prerana1821) [inline components/Footer/Footer.js:38]: Please remove the unwanted code.

File: components/Footer/Footer.js [hunk 2]
```diff
@@ -1,26 +1,114 @@
...
+            <FooterIconLink href="https://twitter.com/neogcamp" imgSrc="/images/twitter.svg" />
+            <FooterIconLink href="https://www.youtube.com/watch?v=Ezk2AwqgS9Q&list=PLzvhQUIpvvuj5KPnyPyWsvgyzNkX_ACPA" imgSrc="/images/youtube.svg" />
+            <FooterIconLink href="https://www.linkedin.com/company/neog-camp/" imgSrc="/images/linkedin.svg" />
+            <FooterIconLink href="https://letters.neog.camp/" imgSrc="/images/substack.svg" />
+            <FooterIconLink href="https://www.facebook.com/neogcamp" imgSrc="/images/facebook.svg" />
+            <FooterIconLink href="https://t.me/teamtanay" imgSrc="/images/telegram.svg" />
+          </div>
+        </footer>
+        <section className="footer-container-other footer-mobile">
+          {/* <div className="mt-1 mb-1 footer links-container-mob">
```
Comment (prerana1821) [inline components/Footer/Footer.js:86]: Please remove the unwanted code.

File: styles/globals.css [hunk 3]
```diff
@@ -1,16 +1,36 @@
...
 html,
 body {
   padding: 0;
   margin: 0;
-  font-family: "Work Sans", sans-serif;
+  font-family: 'Inter', sans-serif;
 }
 
 :root {
-  --primary-color: #7c3aed;
```
Comment (prerana1821) [inline styles/globals.css:11]: Please make sure to remove `--primary-color` wherever it appears.

**Review by prerana1821 — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 3 above)
Comment (pranshudobhal) [reply styles/globals.css:11]: @prerana1821 I've replaced all instances of `--primary-color` with `--primary`.

File: components/Footer/Footer.js (hunk 1 above)
Comment (pranshudobhal) [reply components/Footer/Footer.js]: Removed the unwanted code.

**Pull request merged**